vsphere_plugin_common/__init__.py

```diff
@@ -707,10 +769,32 @@ def get_server_ip(self, vm, network_name):
                 )
                 return ip_address
 
-    def _wait_vm_running(self, task):
+    def _task_guest_state_is_running(self, task):
```

Comment:
On the one hand, yes, they could be. However, they're not used outside that class, so I'd be inclined to leave them there for the moment until we do more serious refactoring of that part of the plugin and figure out how it should be laid out.